### PR TITLE
Create lint-vocab.yml in workflows

### PR DESCRIPTION
**Pull request by iherman**
This is the PR-actionable part of https://github.com/w3c/vc-bitstring-status-list/issues/157#issuecomment-2031760132. Note that extra actions are needed to be done by the maintainer of the repository to ensure that the result of the github actions determine the content of gh-pages.

### CODE REVIEW

**Review by dlongley — APPROVED** [review]
Seems ok to me. I'll note that node 16 is EOL, so node 18 or 20 should be used instead (probably).

File: .github/workflows/lint-vocab.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup environment
+        uses: actions/checkout/@v3
+      - name: Setup Node 16
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
```
Comment (iherman) [inline .github/workflows/lint-vocab.yml:17]:
```suggestion
      - name: Setup Node 20
        uses: actions/setup-node@v3
        with:
          node-version: 20
```

Comment (iherman) [reply .github/workflows/lint-vocab.yml:17]: @dlongley is this what you mean?

Note that this should be updated in other repositories, too!

Comment (dlongley) [reply .github/workflows/lint-vocab.yml:17]: Yeah, that's what I meant.

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
Build process (editorial), multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**